src/site/xdoc/server/config-blob.xml

```diff
@@ -0,0 +1,109 @@
+<?xml version="1.0"?>
+<!--
+  Licensed to the Apache Software Foundation (ASF) under one
+  or more contributor license agreements.  See the NOTICE file
+  distributed with this work for additional information
+  regarding copyright ownership.  The ASF licenses this file
+  to you under the Apache License, Version 2.0 (the
+  "License"); you may not use this file except in compliance
+  with the License.  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing,
+  software distributed under the License is distributed on an
+  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  KIND, either express or implied.  See the License for the
+  specific language governing permissions and limitations
+  under the License.    
+-->
+<document>
+
+ <properties>
+  <title>Apache James Server 3 - WebAdmin Configuration</title>
+ </properties>
+
+<body>
+
+  <section name="Blob Configuration">
+      <p>Consult <a href="https://github.com/apache/james-project/blob/master/dockerfiles/run/guice/cassandra-rabbitmq/destination/conf/blob.properties">blob.properties</a> in GIT to get some examples and hints.</p>
+
+      <subsection name="Blob Store Configuration">
+          <p>
+              <a href="https://github.com/apache/james-project/blob/master/server/blob/blob-api/src/main/java/org/apache/james/blob/api/BlobStore.java">BlobStore</a>
+              is a component used to storing blob data, it can take InputStream or byte array as the input data to store, then return the corresponding
+              <a href="https://github.com/apache/james-project/blob/master/server/blob/blob-api/src/main/java/org/apache/james/blob/api/BlobId.java">BlobId</a>.
+              Clients can use BlobStore to retrieve stored data by the BlobId.
+              <dl>
+                  <dt><strong>implementation</strong></dt>
+                  <dd>
+                      Define the BlobStore implementation will be used in the product. Available implementations: cassandra, objectstorage, union.
+                      cassandra: blobs will be stored and retrieved at cassandra database.
+                      objectstorage: blobs will be stored and retrieved at open-stack swift.
+                      union: blobs will be stored and retrieved at the current BlobStore(open-stack swift), if fails or not found, it fallback to use the legacy BlobStore (cassandra)
+                      Default is cassandra.
+                  </dd>
+              </dl>
+          </p>
+
+          <subsection name="Swift BlobStore Configuration">
+              <dl>
+                  <dt><strong>objectstorage.payload.codec</strong></dt>
+                  <dd>AES256: Aes256 encrypted blob content. DEFAULT: non encrypted</dd>
+
+                  <dt><strong>objectstorage.provider</strong></dt>
+                  <dd>Should be swift</dd>
```

Comment:
if there's only one value, we should not expose it at the configuration level.

Reply:
If I don't specify the compulsory configuration properties. Then the users read this document don't know they have to fill up those properties, otherwise James would not start. And I think it's still neccessary

Reply:
Yes, what I mean is that we have to fix the code if it really requires to provide a single fixed value